tests/acl/test_acl_l3v4v6.py

```diff
@@ -0,0 +1,1224 @@
+                        .format(acl_table_config["table_name"], namespace, duthost))
+            if topo == "t2":
+                acl_table_ports = setup["acl_table_ports"]
+                acl_table_ports = acl_table_ports[duthost]
+                if not len(acl_table_ports[namespace]):
+                    continue
+            else:
+                acl_table_ports = setup["acl_table_ports"]
+            sonic_host_or_asic_inst.command(
+                "config acl add table {} {} -s {} -p {}".format(
+                    acl_table_config["table_name"],
+                    acl_table_config["table_type"],
+                    acl_table_config["table_stage"],
+                    ",".join(acl_table_ports[namespace]),
+                )
+            )
+        else:
+            logger.info("Removing ACL table \"{}\" in namespace {} on device {}"
+                        .format(acl_table_config["table_name"], namespace, duthost))
+            sonic_host_or_asic_inst.command("config acl remove table {}".format(acl_table_config["table_name"]))
+
+
+@pytest.fixture(scope="module")
+def acl_table(duthosts, rand_one_dut_hostname, setup, stage, ip_version, tbinfo):
+    """Apply ACL table configuration and remove after tests.
+
+    Args:
+        duthosts: All DUTs belong to the testbed.
+        rand_one_dut_hostname: hostname of a random chosen dut to run test.
+        setup: Parameters for the ACL tests.
+        stage: The ACL stage under test.
+        ip_version: The IP version under test
+
+    Yields:
+        The ACL table configuration.
+
+    """
+    table_name = "DATA_{}_{}_TEST".format(stage.upper(), ip_version.upper())
+    topo = tbinfo["topo"]["type"]
+
+    acl_table_config = {
+        "table_name": table_name,
+        "table_ports": ",".join(setup["acl_table_ports"]['']),
+        "table_stage": stage,
+        "table_type": "L3V4V6"
+    }
+    logger.info("Generated ACL table configuration:\n{}".format(pprint.pformat(acl_table_config)))
+
+    dut_to_analyzer_map = {}
+
+    for duthost in duthosts:
+        if duthost.is_supervisor_node():
+            continue
+        statedb_capability = \
+            'sudo sonic-db-cli STATE_DB HGET "ACL_STAGE_CAPABILITY_TABLE|{}" supported_L3V4V6'.format(stage.upper())
+        supported = duthost.shell(statedb_capability)['stdout'].strip()
+        logger.info("[{}]=>{}.".format(statedb_capability, supported))
+        if supported != 'true':
+            pytest.skip("L3V4V6 ACL not supported in stage {}.".format(stage))
+        loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix="acl")
+        loganalyzer.load_common_config()
+        dut_to_analyzer_map[duthost] = loganalyzer
+
+        try:
+            loganalyzer.expect_regex = [LOG_EXPECT_ACL_TABLE_CREATE_RE]
+            # Ignore any other errors to reduce noise
+            loganalyzer.ignore_regex = [r".*"]
+            with loganalyzer:
```

Comment:
You may not need to check syslog. There is status for each ACL table. The table is created successfully if the status is `active`.